src/main/java/it/gov/pagopa/admissibility/service/onboarding/OnboardingRequestEvaluatorServiceImpl.java

```diff
@@ -26,7 +26,7 @@ public OnboardingRequestEvaluatorServiceImpl(RuleEngineService ruleEngineService
     public Mono<EvaluationDTO> evaluate(OnboardingDTO onboardingRequest, InitiativeConfig initiativeConfig) {
         final EvaluationDTO result = ruleEngineService.applyRules(onboardingRequest, initiativeConfig);
 
-        if(OnboardingConstants.ONBOARDING_STATUS_OK.equals(result.getStatus())){
+        if(OnboardingConstants.ONBOARDING_STATUS_OK.equals(result.getStatus()) && !Boolean.TRUE.equals(initiativeConfig.getRankingInitiative())){
```

Comment:
maybe it could be more usefull to let the ruleEngineService to build the EvaluationDto to be send to outcome or to the ranking system.
This will simplify the mediator :)